### PR TITLE
Update to @zippytech/react-load-mask v4

### PR DESCRIPTION
**Pull request by jfsiii**
Had to use github link in package.json b/c of https://github.com/zippytech/react-load-mask/issues/1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,13 +48,15 @@
     "webpack-dev-server": "^1.8.2"
   },
   "dependencies": {
+    "@zippytech/react-load-mask": "https://github.com/zippytech/react-load-mask",
     "drag-helper": "^1.2.3",
     "has-touch": "^1.0.1",
     "object-assign": "^4.1.0",
+    "prop-types": "^15.5.10",
+    "react": "^15.6.2",
```
Comment (jfsiii) [inline package.json]: Didn't mean to include this. Will update soon.

File: package.json [hunk 2]
```diff
@@ -48,13 +48,15 @@
...
     "react-class": "^2.1.0",
-    "react-load-mask": "^2.0.1",
-    "react-style-normalizer": "^1.2.7",
-    "prop-types": "^15.5.10"
+    "react-dom": "^15.6.2",
```
Comment (jfsiii) [inline package.json]: Didn't mean to include this. Will update soon.

**Comment by jfsiii** [comment]
@stevelacy this is ready now.

**Comment by stephenlacy** [comment]
published to 2.3.0

**Pull request merged**